app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -287,6 +287,7 @@ def sync_complete
     def remove_content
       fail _("No content ids provided") if @content.blank?
       respond_for_async :resource => sync_task(::Actions::Katello::Repository::RemoveContent, @repository, @content)
+      async_task(::Actions::Katello::Repository::CapsuleGenerateAndSync, @repository)
```

Comment:
Yeah, I would agree, in addition we should likely do something similar to https://github.com/Katello/katello/pull/6544 so that during a multiple file upload the repository isn't synced for every single file.